headers/app.py

```diff
@@ -0,0 +1,24 @@
+# Driver for ML component of MarkLogic Classifier.
+import sys
+import classifier as cfr
+
+# Params: ontology = JSON object passed by frontend
+#         arg = list of CSV file paths
+# Returns: The classification metadata
+# TODO: More clearly specify what metadata to return
+def classify(ontology, *arg):
+    # Verify parameters
+    if len(arg) == 0:
+        print_usage()
+    my_classifier = cfr.Classifier(arg)
+    results = my_classifier.classify_ontology(ontology)
+    return results
```

Comment:
return my_classifier.classify_ontology(ontology)

Reply:
I think it's fine the way it is. It's for code readability.